source/particle/world.cc

```diff
@@ -1215,6 +1226,7 @@ namespace aspect
       }
       prm.leave_subsection ();
 
+      Generator::Interface<dim>::declare_parameters(prm);
```

Comment:
This call should probably come from the child classes of the Interface that need these parameters. Here it looks a bit out of place.